flit/init.py

```diff
@@ -0,0 +1,178 @@
+import configparser
+import json
+import os
+from pathlib import Path
+import re
+import sys
+
+def get_data_dir():
+    """Get the directory path for flit user data files.
+    """
+    home = os.path.realpath(os.path.expanduser('~'))
+
+    if sys.platform == 'darwin':
+        d = Path(home, 'Library')
+    elif os.name == 'nt':
+        appdata = os.environ.get('APPDATA', None)
+        if appdata:
+            d = Path(appdata)
+        else:
+            d = Path(home, 'AppData', 'Roaming')
+    else:
+        # Linux, non-OS X Unix, AIX, etc.
+        xdg = os.environ.get("XDG_DATA_HOME", None)
+        d = Path(xdg) if xdg else Path(home, '.local/share')
+
+    return d / 'flit'
+
+def get_defaults():
+    try:
+        with (get_data_dir() / 'init_defaults.json').open() as f:
+            return json.load(f)
+    except FileNotFoundError:
+        return {}
+
+def store_defaults(d):
+    data_dir = get_data_dir()
+    try:
+        data_dir.mkdir(parents=True)
+    except FileExistsError:
+        pass
+    with (data_dir / 'init_defaults.json').open('w') as f:
+        json.dump(d, f, indent=2)
+
+license_names_to_classifiers = {
+    'BSD': 'License :: OSI Approved :: BSD License',
+    'GPL': 'License :: OSI Approved :: GPL License',
+    'Apache': 'License :: OSI Approved :: Apache License'
+}
```

Comment:
Sure, this can be added to. I was thinking of following Github's [choosealicense.com](http://choosealicense.com/), which keeps it simple by only offering three options. I misremembered which three, though - they go for MIT instead of BSD.